components.yaml

```
@@ -65,43 +65,69 @@ g5pert:
 GEOSagcmPert_GridComp:
   local: ./src/Components/@GEOSagcmPert_GridComp
   remote: ../GEOSagcmPert_GridComp.git
-  tag: v1.1.0
+  tag: rt-GEOSadas-5_29_0_prep3
   develop: main
 
 FVdycoreCubed_GridComp:
   local: ./src/Components/@GEOSgcm_GridComp/GEOSagcm_GridComp/GEOSsuperdyn_GridComp/@FVdycoreCubed_GridComp
   remote: ../FVdycoreCubed_GridComp.git
-  tag: v1.0.9
+  tag: v1.2.13
   develop: develop
 
 fvdycore:
   local: ./src/Components/@GEOSgcm_GridComp/GEOSagcm_GridComp/GEOSsuperdyn_GridComp/@FVdycoreCubed_GridComp/@fvdycore
   remote: ../GFDL_atmos_cubed_sphere.git
-  tag: geos/v1.0.2
+  tag: geos/v1.1.6
   develop: geos/develop
 
 GEOSchem_GridComp:
   local: ./src/Components/@GEOSgcm_GridComp/GEOSagcm_GridComp/GEOSphysics_GridComp/@GEOSchem_GridComp
   remote: ../GEOSchem_GridComp.git
-  tag: v1.1.1
+  tag: v1.6.0
+  develop: develop
+
+HEMCO:
+  local: ./src/Components/@GEOSgcm_GridComp/GEOSagcm_GridComp/GEOSphysics_GridComp/@GEOSchem_GridComp/Shared/HEMCO/@HEMCO
+  remote: ../HEMCO.git
+  tag: geos/v2.2.1
+  develop: geos/develop 
+
+geos-chem:
+  local: ./src/Components/@GEOSgcm_GridComp/GEOSagcm_GridComp/GEOSphysics_GridComp/@GEOSchem_GridComp/GEOSCHEMchem_GridComp/@geos-chem
+  remote: ../geos-chem.git
+  tag: geos/v13.0.0-rc1
+  develop: geos/develop
+
+GOCART:
+  local: ./src/Components/@GEOSgcm_GridComp/GEOSagcm_GridComp/GEOSphysics_GridComp/@GEOSchem_GridComp/@GOCART
+  remote: ../GOCART.git
+  sparse: ./config/GOCART.sparse
+  tag: v1.0.1
   develop: develop
 
 mom:
   local: ./src/Components/@GEOSgcm_GridComp/GEOSogcm_GridComp/GEOSocean_GridComp/GuestOcean_GridComp/MOM_GEOS5PlugMod/@mom
-  remote: ../MOM5.git
-  tag: geos/v1.0.1
+  remote: git@github.com:GEOS-ESM/MOM5.git
+  tag: geos/5.1.0+1.1.1
   develop: geos5
 
+mom6:
+  local: ./src/Components/@GEOSgcm_GridComp/GEOSogcm_GridComp/GEOSocean_GridComp/GuestOcean_GridComp/MOM6_GEOSPlug/@mom6
+  remote: ../MOM6.git
+  tag: geos/v2.0.1
+  develop: dev/gfdl
+  recurse_submodules: True
+
 GEOSgcm_App:
   local: ./src/Applications/@GEOSgcm_App
   remote: ../GEOSgcm_App.git
-  tag: cvs/GEOSadas-5_27_1_p3
+  tag: rt-GEOSadas-5_29_0_prep3
```

```suggestion
  tag: v1.5.2
```

This should become v1.5.2 which is the semver tag containing the ADAS fixes from @rtodling . Note that this requires updating to MAPL v2.7.0 due to an interface change:

https://github.com/GEOS-ESM/GEOSgcm_App/compare/v1.5.2..rt-GEOSadas-5_29_0_prep3